src/main/java/bio/terra/janitor/common/CredentialUtils.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.janitor.common;
+
+import bio.terra.janitor.common.exception.JanitorCredentialException;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+
+/** Utilities for handling credentials. */
+public class CredentialUtils {
+  /** Gets the {@link ServiceAccountCredentials} from a file path. */
+  public static ServiceAccountCredentials getGoogleCredentialsOrDie(String serviceAccountPath) {
```

Comment:
Do we need this in main since we're setting up environment variables to make the service account use the default google credentials? I see the utility of this in test where we have client service accounts, so let's move it to src/test?

Reply:
Done